language/move-lang/tests/move_check/parser/function_visibility_script.move

```diff
@@ -0,0 +1,5 @@
+module M {
+    public(script) fun f() {}
+    public (script) fun g() {}
+    public ( script ) fun h() {}
```

Comment:
Do we want to parse this? Maybe its okay and we will just clean it up with an autoformatter... some day

Reply:
I guess it is syntactically OK and hence, may be accepted by the parser? The same applies to other languages as well and it is usually the formatter's job to discourage/correct these bad practices.

Reply:
I agree with @mengxu-fb -- this is also consistent with how the Move parser handles other similar cases. Whitespace between tokens should not significant.